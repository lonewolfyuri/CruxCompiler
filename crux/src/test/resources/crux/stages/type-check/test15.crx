// invalid array

array x : void[3];
  
func main() : void {
  ::println();
}
