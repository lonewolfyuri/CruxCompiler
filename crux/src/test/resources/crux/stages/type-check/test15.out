Error type-checking file.
TypeError(3,0)[Array x has invalid base type void.]
